### ISSUE TITLE
Increased batch size for replicating DINOv2

### ISSUE THREAD

**Issue by davnords** [RobinWu218/SimDINO#30]
Hi there, 

I love your work and tried to replicate the ViT-L/16 and ViT-B/16 training of DINOv2 you have done in this repo.

However, after trying to use the hyperparameters you provide in the Appendix I find the loss is quite unstable for DINOv2 training. I will upload the .yaml file I used (as .txt because github does not support yaml).

I used 128x8 batch size as you specify in the Appendix. Did you experience issues with loss going up during parts of training? Should I increase the effective batch size to have better convergence? 

Thank you very much for your patience!

/ David

<img width="640" alt="Image" src="https://github.com/user-attachments/assets/f038592f-8291-4fb6-9e91-d1ed7fe3baf6" />

[ssl_default_config.txt](https://github.com/user-attachments/files/20003965/ssl_default_config.txt)

**Comment by davnords** [comment]
For reference I found this in the DINOv2 repo, so perhaps I should not monitor the loss too closely, i am not sure :D 

<img width="919" alt="Image" src="https://github.com/user-attachments/assets/26c49a94-6afc-48ce-a870-fd837d0aaabe" />

**Comment by davnords** [comment]
I'll close this issue because i can verify that, while seemingly very stochastic, it actually learns stuff when you perform linear / knn evaluation so I think the configs you provided should work fine.
